queries/roc/highlights.scm

```diff
@@ -0,0 +1,228 @@
+;----comments----
+[
+  (line_comment)
+  (doc_comment)
+] @comment
+
+;-----Punctuation----
+[
+  "?"
+  (arrow)
+  (back_arrow)
+  (backslash)
+] @punctuation.delimiter
+
+[
+  ","
+  ":"
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+] @punctuation.bracket
+
+[
+  "|"
+  "&"
+  (operator)
+  (wildcard_pattern)
+] @operator
+
+[
+  "if"
+  "then"
+  "else"
+] @conditional
+
+[
+  (implements)
+  (when)
+  (is)
+  "as"
+  (to)
+] @keyword.control.roc
+
+;----headers-----
+(interface_header
+  (name) @type.interface)
+
+(imports
+  (imports_entry
+    (module) @namespace))
+
+(packages
+  (record_pattern
+    (record_field_pattern
+      (field_name) @namespace)))
+
+(app_name) @string
+
+(import_path) @string
+
+[
+  "app"
+  "packages"
+  "imports"
+  "provides"
+  "interface"
+  "exposes"
+  "expect"
+  (import_as)
+] @keyword.control
+
+;---annotations----
+(annotation_type_def
+  (annotation_pre_colon
+    (identifier) @type)
+  (#set! "priority" 99))
+
+(annotation_type_def
+  (annotation_pre_colon
+    (identifier) @function)
+  (function_type))
+
+;----decleration types----
+(value_declaration
+  (decl_left
+    (identifier_pattern
+      (identifier) @parameter.definition))
+  (#set! "priority" 99))
+
+(backpassing_expr
+  assignee: (identifier_pattern
+    (identifier) @parameter.definition))
```

Comment:
Captures doesn't seem nvim-treesitter like
Please follow CONTRIBUTING.md guideline on allowed captures

Reply:
I have read that document and I do not understand what I need to change here.

Reply:
`@parameter.definition` is not an allowed capture name. You can **only** use the captures listed in CONTRIBUTING.md, nothing else.